### ISSUE TITLE
Path not found error during installation 

### ISSUE THREAD

**Issue by SB21BCB** [flatironinstitute/CaImAn#634]
For better support, please use the template below to submit your issue. When your issue gets resolved please remember to close it.

Sometimes errors while running CNMF occur during parallel processing which prevents the log to provide a meaningful error message. Please reproduce your error with setting `dview=None`.

If you need to upgrade CaImAn follow the instructions given in the [documentation](https://caiman.readthedocs.io/en/master/Installation.html#upgrading).

- **Tell us a bit about your setup**: 
 1. Operating system (Linux/macOS/Windows): 
 Windows 10 Enterprise
 2. Python version (3.x): 
 3.7
 3. Working environment (Python IDE/Jupyter Notebook/other):
 Spyder IPython console
 4. Which of the demo scripts you're using for your analysis (if applicable):
 N/A
 5. CaImAn version*: 
 Not yet installed
 6. CaImAn installation process (`pip install .`/`pip install -e .`/conda): 
Have not gotten that far
*You can get the CaImAn version by creating a `params` object and then typing `params.data['caiman_version']`. If the field doesn't exist, type N/A and consider upgrading)

- **Describe the issue that you are experiencing*
When running "conda activate caiman" command I am getting a path not found error.
No errors during installation to this point.

- **Copy error log below**
conda activate caiman

C:\Users\sdrulla\CaImAn>SET DISTUTILS_USE_SDK=1 

C:\Users\sdrulla\CaImAn>SET MSSdk=1 

C:\Users\sdrulla\CaImAn>SET "VS_VERSION=15.0" 

C:\Users\sdrulla\CaImAn>SET "VS_MAJOR=15" 

C:\Users\sdrulla\CaImAn>SET "VS_YEAR=2017" 

C:\Users\sdrulla\CaImAn>set "MSYS2_ARG_CONV_EXCL=/AI;/AL;/OUT;/out" 

C:\Users\sdrulla\CaImAn>set "MSYS2_ENV_CONV_EXCL=CL" 

C:\Users\sdrulla\CaImAn>set "PY_VCRUNTIME_REDIST=\\bin\\vcruntime140.dll" 

C:\Users\sdrulla\CaImAn>set "VSINSTALLDIR=" 

C:\Users\sdrulla\CaImAn>if exist "C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe" (for /F "usebackq tokens=*" %i in (`"C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe" -nologo -products * -version [15.0,16.0) -property installationPath`) do (set "VSINSTALLDIR=%i\" ) ) 

C:\Users\sdrulla\CaImAn>if not exist "" (set "VSINSTALLDIR=C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\" ) 

C:\Users\sdrulla\CaImAn>if not exist "C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\" (set "VSINSTALLDIR=C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\" ) 

C:\Users\sdrulla\CaImAn>if not exist "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\" (set "VSINSTALLDIR=C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\" ) 

C:\Users\sdrulla\CaImAn>if not exist "C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\" (set "VSINSTALLDIR=C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\" ) 

C:\Users\sdrulla\CaImAn>IF NOT "" == "" (
set "INCLUDE=;"  
 set "LIB=;"  
 set "CMAKE_PREFIX_PATH=;" 
) 

C:\Users\sdrulla\CaImAn>pushd "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\"  

C:\Users\sdrulla\CaImAn>CALL "VC\Auxiliary\Build\vcvars64.bat"  

C:\Users\sdrulla\CaImAn>popd  

C:\Users\sdrulla\CaImAn>IF "" == "" (SET "CMAKE_GENERATOR=Visual Studio 15 2017 Win64"  ) 

Note: you may need to restart the kernel to use updated packages.
The system cannot find the path specified.
The system cannot find the path specified.

- **If you're not reporting an error, type your message below**

**Comment by pgunn** [comment]
Hi,
In the instructions we tell you to delete a vs-related file from the conda directory. These errors come from not doing that.

**Comment by SB21BCB** [comment]
Hello Pat and thank you for the response.  I read the instructions 

Use the Windows find-file utility (under the Start Menu) to look for vs2015_compiler_vars.bat and/or vs2015_compiler_vars.bat under your home directory. At least one copy should show up. Delete the version that has conda:raw-latex:envs:raw-latex:`\caiman` <https://github.com/flatironinstitute/CaImAn/tree/master/docs/source#id1> as part of its location.

When I searched, there were no file with these names found.  If I search the conda directory for any vs* files and delete them, would this be sufficient?

Thanks,
Sean

**Comment by pgunn** [comment]
Yup, give that a look. Sorry for the terseness of my responses right now - working from my phone.

**Comment by SB21BCB** [comment]
Not a problem. Thanks and have a good weekend. 

-- Sean
